internal/testutils/feature.go

```diff
@@ -48,13 +48,13 @@ func CheckFeatureTest(t *testing.T, fn func() error) {
 }
 
 func SkipIfNotSupported(tb testing.TB, err error) {
-	ufe, ok := errors.Cause(err).(*internal.UnsupportedFeatureError)
-	if !ok {
-		return
+	for err != nil {
+		if ufe, ok := err.(*internal.UnsupportedFeatureError); ok {
```

Comment:
This won't work if the error is wrapped. I think you need to use xerrors.As instead.